scripts/solr_builder/solr_builder/solr_builder.py

```diff
@@ -1,6 +1,6 @@
 import json
```

Comment:
```suggestion
from __future__ import annotations

import json
```
This might magically fix our failing GitHub Action.

Reply:
I will add this `__future__` import and then (as we discussed on Slack) look at the [commits](https://github.com/internetarchive/openlibrary/pull/6826/commits) to see all the changes that `pre-commit` + `pyupgrade` make based on it’s presence.  Does that automatic commit fix our failing test?

Remember to `git pull` these changes to your local branch before making any further changes.